pocApp/new_tasks.py

```diff
@@ -0,0 +1,126 @@
+            scripts = host.getElementsByTagName("script")
+            for script in scripts:
+                for elem in script.getElementsByTagName("elem"):         # Get cert details for each target 
+                    try:
+                        if elem.getAttribute("key") == 'commonName':
+                            if commonName == '':                            # Only get the first commonName 
+                                commonName =  elem.childNodes[0].nodeValue
+                    except:
+                        pass
+                    try:
+                        if elem.getAttribute("key") == 'organizationName':
+                            if organizationName == '': 
+                               organizationName =  elem.childNodes[0].nodeValue
+                    except:
+                        pass
+                    try:
+                        if elem.getAttribute("key") == 'countryName':
+                           countryName =  elem.childNodes[0].nodeValue
+                    except:
+                        pass
+                
+                    try:
+                        if elem.getAttribute("key") == 'notBefore':
+                            notBefore =  elem.childNodes[0].nodeValue
+                            notBefore = notBefore.split('T')[0]
+                    except:
+                        pass
+                    try:
+                        if elem.getAttribute("key") == 'notAfter':
+                            notAfter =  elem.childNodes[0].nodeValue
+                            notAfter = notAfter.split('T')[0]
+                    except:
+                        pass
+
+                    try:
+                        if elem.getAttribute("key") == 'md5':
+                            md5 =  elem.childNodes[0].nodeValue
+                            md5 =  md5.split('T')[0]
+                    except:
+                        pass
+
+                    try:
+                        if elem.getAttribute("key") == 'sha1':
+                            sha1 =  elem.childNodes[0].nodeValue
+                            sha1 =  sha1.split('T')[0]
+                    except:
+                        pass
+
+                    #try:
+                    #    if elem.getAttribute("key") == 'commonName':
+                    #        host_name =  elem.childNodes[0].nodeValue
+                    #        host_name =  host_name.split('T')[0]
+                    #except:
+                    #    pass
+            
+                #output.append(ip + ',' + commonName + ',' + organizationName + ',' + countryName + ',' + 
+                #notBefore + ',' + notAfter)
+                output.append(commonName + ":" + notBefore + ',' + notAfter + ", " + host_name + ", " +  md5 + ',' + sha1)
+                logger.info("output: ", output)
+                print("Hello:-----------------------",output)
+                try:
+                    dbCerts=MyCertsData()
```

Comment:
we need to come up with a better way of handling the data. please hang a story on the backlog about ssl_certs history